### ISSUE TITLE
Provide an easier way to show untagged data

### ISSUE THREAD

**Issue by edgimar** [nomeata/arbtt#23]
Currently the way to write a categorize.cfg file is tedious, involving the use of `--dump-samples` in combination with several `--exclude`  commandline arguments, and piping this through various GNU shell tools.  A better way would be to have a command/option within categorize.cfg which allows one to tag only those samples that haven't been otherwise tagged.

One way of accomplishing this could be to have a special variable/condition which matches the "leftover" samples, after all other rules have been applied, so you could have the rule look like `$not_otherwise_tagged ==> tag Untagged:$current.program:$title`  (it would only make sense for such a rule to occur once in the config file).

If this were possible, then creating a categorize.cfg file would be much faster because you could simply run arbtt-stats with your config-file, and every time you added a rule that matched some samples, then the output of arbtt-stats would tell you what things still haven't been matched.  After you're satisfied with the categorization, the "untagged" rule could be removed/disabled.

**Comment by nomeata** [comment]
Nice idea. It would only work if people use tags along one axis, right? So if I always tag `weekday` and `weekend`, then everything is tagged, but that is not what you want, right?

It seems that this is related to categories: if you use categories to tag your samples, then you probably want to know all samples that are not categorized according to a specific category. And that is easy: `-x Work:` gives you all samples that do not have any `Work:something` tag attached.

**Comment by edgimar** [comment]
Yes, that could work -- I hadn't thought to use categories for this.  It might still be nice to be able to specify an equivalent to the `-x` flag that can be specified in the config file, though.
